src/Microsoft.PowerShell.Commands.Management/commands/management/SetClipboardCommand.cs

```diff
@@ -98,21 +141,21 @@ protected override void ProcessRecord()
             if (Value == null && _isHtmlSet)
             {
                 ThrowTerminatingError(new ErrorRecord(new InvalidOperationException(
-    string.Format(CultureInfo.InvariantCulture, ClipboardResources.InvalidHtmlCombine)),
-    "FailedToSetClipboard", ErrorCategory.InvalidOperation, "Clipboard"));
+                    string.Format(CultureInfo.InvariantCulture, ClipboardResources.InvalidHtmlCombine)),
+                    "FailedToSetClipboard", ErrorCategory.InvalidOperation, "Clipboard"));
```

Comment:
FQEID

Reply:
This is an existing FQEID, so it would be a breaking change